classes/admin.php

```diff
@@ -745,7 +745,11 @@ public function getPage($path)
                 // Found the type and header from the session.
                 $data = $this->session->{$page->route()};
 
-                $header = ['title' => $data['title']];
+                // Initialise the page headers with the default page headers
+                $default_headers = $this->grav['config']->get('plugins.admin.editor.default_headers');
+                $header = Yaml::parse($default_headers);
+
+                $header['title'] = $data['title'];
```

Comment:
I'd prefer to have the title before the default page headers. It has become kind of standard to always have the `title` as the first entry of a frontmatter. It also feels more natural to have the user defined entries be after the core ones.

This could potentially also allow to override the title, even though it would be crazy to do so, there might be some scenarios out there, where this might be desired. 😄